addons/account/views/account_move_views.xml

```diff
@@ -373,7 +373,7 @@
                         <filter string="Matching" name="group_by_matching" domain="[]" context="{'group_by': 'matching_number'}"/>
                     </group>
                     <searchpanel class="account_root w-auto">
-                        <field name="account_root_id" icon="fa-filter" groupby="account_id" limit="0"/>
```

Comment:
According to the ticket (and my test), this one still works as expected, any idea why?